sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -3204,12 +3284,12 @@
                     "calculateServer": false,
                     "allowCalculateOverride": false,
                     "validate": {
-                        "required": true,
+                        "required": false,
```

Comment:
isn't this question required?

Reply:
It has a custom validation. If we keep both the "required" validation is triggered instead of the custom one.

Reply:
so, just wondering if we make it false, won't that red `*` disappear?

Reply:
Yes, you're right. I removed the required part from the custom logic and reverted the "required" property to be false. Now both work and there's the red asterisk back. Thanks!

Reply:
I'm confused here. Required and custom validation both can happen together right?

like this. Let me know if I am missing something specific to your case.
![image](https://github.com/bcgov/SIMS/assets/54600590/ee01f2af-ab59-4b72-92fe-d6e83e916493)

Reply:
Never mind. Ignore my previous comment.